ons_alpha/core/models/snippets.py

```diff
@@ -99,3 +102,25 @@ class Meta:
 
     def __str__(self):
         return str(self.name)
+
+
+@register_snippet
+class Chart(index.Indexed, PreviewableMixin, models.Model):
+    chart = StreamField([("chart", ChartBlock())], use_json_field=True, min_num=1, max_num=1)
```

Comment:
I could swear there's a nicer way of doing this, I just forget what it is. For now, this does work, it just requires clicking an extra "+" to show the inputs.